egs/wham/ConvTasNet/local/prepare_data.sh

```diff
@@ -24,8 +25,9 @@ wait
 echo "Run python scripts to create the WHAM mixtures"
 # Requires : Numpy, Scipy, Pandas, and Pysoundfile
 cd $out_dir/wham_scripts
-$python_path create_wham_from_scratch.py \
+ln -sf ../../../corpus .
```

Comment:
Why this? 
This looks like a personal path, right?

Reply:
Sorry will fix this one.